### PR TITLE
Update to 0.11

### PR DESCRIPTION
**Pull request by mariusvniekerk**

### CODE REVIEW

**Review by xhochy — APPROVED** [review]

**Comment by wesm** [comment]
Have we tested parquet-cpp against Thrift 0.11? I believe there are some API changes

**Comment by wesm** [comment]
I thought I saw `boost::shared_ptr` replaced with `std::shared_ptr`, but I might be wrong

**Comment by mariusvniekerk** [comment]
Looking at https://github.com/apache/thrift/blob/master/lib/cpp/src/thrift/stdcxx.h it seems that way @wesm .  We can possibly build with different compiler flags if needed?

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
   name: thrift-cpp
-  version: "0.10.0"
+  version: "0.11.0"
 
 source:
-  fn: thrift-0.10.0.tar.gz
-  url: http://archive.apache.org/dist/thrift/0.10.0/thrift-0.10.0.tar.gz
-  md5: 795c5dd192e310ffff38cfd9430d6b29
+  fn: thrift-0.11.0.tar.gz
+  url: http://archive.apache.org/dist/thrift/0.11.0/thrift-0.11.0.tar.gz
```
Comment (jakirkham) [inline recipe/meta.yaml:7]: Might be worth templating these using Jinja.

**Comment by wesm** [comment]
Here's a PR upgrading parquet-cpp https://github.com/apache/parquet-cpp/pull/433

**Comment by wesm** [comment]
Go ahead and merge this whenever it's ready

**Comment by wesm** [comment]
This is quite the Travis CI backlog! I'm going to go ahead and merge this in the hopes of getting the packages updated as soon as possible

**Pull request merged**